multiqc/modules/slamdunk/slamdunk.py

```diff
@@ -0,0 +1,832 @@
+        
+        self.tc_per_utrpos_plus = dict()
+        self.tc_per_utrpos_minus = dict()
+
+        # Switch string for plots to produce
+        produce = "" 
+
+        # Check whether summary also contains number of counted reads
+        extendedSummary = True
+            
+        for f in self.find_log_files(config.sp['slamdunk']['summary'], filehandles = True):
+            extendedSummary = extendedSummary and self.parseSummary(f)
+            
+        if len(self.slamdunk_data) == 0:
+            log.debug("Could not find any reports in {}".format(config.analysis_dir))
+        else :
+            produce += "summary"
+            
+        if extendedSummary:
+            log.info("Extended Slamdunk summary found.")
+            
+        log.info("Parsing PCA reports.")
+            
+        for f in self.find_log_files(config.sp['slamdunk']['PCA'], filehandles = True):
+            self.parsePCA(f)
+            
+        if len(self.PCA_data) == 0:
+            log.debug("Could not find PCA reports in {}".format(config.analysis_dir))
+        else :
+            produce += "PCA"
+            
+        log.info("Parsing UTR rates reports. This can take a while...")
+                        
+        for f in self.find_log_files(config.sp['slamdunk']['utrrates'], filehandles = True):
+            self.parseUtrRates(f)
+            
+        if len(self.utrates_data) == 0:
+            log.debug("Could not find UTR rates reports in {}".format(config.analysis_dir))
+        else :
+            produce += "utrates"
+            
+
+        log.info("Parsing read rates reports.")
+            
+        for f in self.find_log_files(config.sp['slamdunk']['rates'], filehandles = True):
+            self.parseSlamdunkRates(f)
+            
+        if len(self.rates_data_plus) == 0:
+            log.debug("Could not find read rates reports in {}".format(config.analysis_dir))
+        else :
+            produce += "readrates"
+            
+        log.info("Parsing rates per read position reports.")
+            
+        for f in self.find_log_files(config.sp['slamdunk']['tcperreadpos'], filehandles = True):
+            self.parseSlamdunkTCPerReadpos(f)
+            
+        if len(self.tc_per_readpos_plus) == 0:
+            log.debug("Could not find conversion per read position reports in {}".format(config.analysis_dir))
+        else :
+            produce += "readpos"
+            
+        log.info("Parsing rates per UTR position reports.")
+
+        for f in self.find_log_files(config.sp['slamdunk']['tcperutrpos'], filehandles = True):
+            self.parseSlamdunkTCPerUtrpos(f)
+            
+        if len(self.nontc_per_utrpos_plus) == 0:
+            log.debug("Could not find conversion per UTR position reports in {}".format(config.analysis_dir))
+        else :
+            produce += "utrpos"
+        
+        if (produce == "") :
+            log.info("No slamdunk reports found.")
```

Comment:
This will show for every user who ever runs MultiQC even if they've never heard of slamdunk. The log would be way too long to read if all modules did this - please convert to a `log.debug`.